pyfr/solvers/navstokes/kernels/bcs/slp-adia-wall.mako

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%include file='pyfr.solvers.euler.kernels.rsolvers.${rsolver}'/>
+<%include file='pyfr.solvers.navstokes.kernels.bcs.common'/>
+<%include file='pyfr.solvers.navstokes.kernels.flux'/>
+
+<%pyfr:macro name='bc_common_flux_state' params='ul, gradul, nl, magnl'>
+    // Ghost state r
+    fpdtype_t ur[${nvars}];
+
+    fpdtype_t nor = ${' + '.join('ul[{1}]*nl[{0}]'.format(i, i + 1)
```

Comment:
I can see bcconu needs bc_ldg_state, but the code is still running(perhaps with erroneous results (or not!), I will need time to make that claim).  Just to figure out where is bcconu getting called?